### PR TITLE
fix #56041 passnavi.evidus.com

### LINKED ISSUE #56041: passnavi.evidus.com
### Issue URL (Incorrect Blocking)
 [https://passnavi.evidus.com/search_univ/0220/ippan.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fpassnavi.evidus.com%2Fsearch_univ%2F0220%2Fippan.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>

### System configuration

Information | value
--- | ---
Platform: | And 10
AdGuard version: | 3.5 nightly 1 (5)
Browser: | Chrome (data compression)
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | enabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/56041

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -22,6 +22,7 @@
 ||nd.demdex.net/event?d_mid=$empty,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$empty,important,domain=9now.com.au
 !
+||passnavi.evidus.com/js/*_tracking
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:25]: Is everything works fine with this rule?

Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:25]: Also missed tracker:
`||track.edu-info.jp^`

 And broken redirect
https://passnavi.evidus.com/ct.html?ad=tokai&page=special_plan&number=1&uri=/sp/tokai
`@@||google-analytics.com/analytics.js$domain=passnavi.evidus.com`

Comment (Yuki2718) [reply SpywareFilter/sections/specific.txt:25]: Looks fine without logged in, I'll test with logged in also with that rule added. For the redirect
`||google-analytics.com/analytics.js$script,redirect=google-analytics,domain=passnavi.evidus.com`
works for supported platforms.

Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:25]: Ok, added in my commit.

**Pull request closed without merging**